g3doc/OpDefinitions.md

```diff
@@ -18,7 +18,7 @@ best practices. They are rendered as quoted bullet points.
 ## Motivation
 
 MLIR allows pluggable dialects, and dialects contain, among others, a list of
-operations. This open and extensible ecosystem leads to the "stringly" type IR
+operations. This open and extensible ecosystem leads to the "strongly" type IR
```

Comment:
stringly was intended here.

Reply:
Thank you for pointing out. I should have read the next line carefully.